src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -436,8 +436,25 @@ def run_single_testcase(self,
                           testcase_path,
                           timeout=None,
                           additional_args=None):
-    # TODO(flowerhack): Fill out this command.
-    pass
+    # TODO(flowerhack): a lot of this code is repetitive with `fuzz`; once we
+    # transition to the new fuzzing interface, a lot of this should be
+    # factored out.
+    # Copy the testcase from the host to the target.
+    testcase_path_name = os.path.basename(os.path.normpath(testcase_path))
+    self.device.store(testcase_path, self.fuzzer.data_path())
+    # Run a single round, using the testcase we just pushed to target.
+    self.fuzzer.start([self.fuzzer.data_path(testcase_path_name)])
```

Comment:
as discussed, you need to pipe additional_args into the fuzzer CLI. You also need to fix this in fuzz().